Explorer/Assets/DCL/SDKComponents/SceneUI/Systems/UITransformHandlerSystem.cs

```diff
@@ -0,0 +1,180 @@
+
+            if (model.MinWidthUnit != YGUnit.YguUndefined)
+                visualElementToSetup.style.minWidth = new Length(model.MinWidth, UiElementUtils.GetUnit(model.MinWidthUnit));
+
+            // Paddings
+            if (model.PaddingBottomUnit != YGUnit.YguUndefined)
+                visualElementToSetup.style.paddingBottom = new Length(model.PaddingBottom, UiElementUtils.GetUnit(model.PaddingBottomUnit));
+
+            if (model.PaddingLeftUnit != YGUnit.YguUndefined)
+                visualElementToSetup.style.paddingLeft = new Length(model.PaddingLeft, UiElementUtils.GetUnit(model.PaddingLeftUnit));
+
+            if (model.PaddingRightUnit != YGUnit.YguUndefined)
+                visualElementToSetup.style.paddingRight = new Length(model.PaddingRight, UiElementUtils.GetUnit(model.PaddingRightUnit));
+
+            if (model.PaddingTopUnit != YGUnit.YguUndefined)
+                visualElementToSetup.style.paddingTop = new Length(model.PaddingTop, UiElementUtils.GetUnit(model.PaddingTopUnit));
+
+            // Margins
+            if (model.MarginLeftUnit != YGUnit.YguUndefined)
+                visualElementToSetup.style.marginLeft = new Length(model.MarginLeft, UiElementUtils.GetUnit(model.MarginLeftUnit));
+
+            if (model.MarginRightUnit != YGUnit.YguUndefined)
+                visualElementToSetup.style.marginRight = new Length(model.MarginRight, UiElementUtils.GetUnit(model.MarginRightUnit));
+
+            if (model.MarginBottomUnit != YGUnit.YguUndefined)
+                visualElementToSetup.style.marginBottom = new Length(model.MarginBottom, UiElementUtils.GetUnit(model.MarginBottomUnit));
+
+            if (model.MarginTopUnit != YGUnit.YguUndefined)
+                visualElementToSetup.style.marginTop = new Length(model.MarginTop, UiElementUtils.GetUnit(model.MarginTopUnit));
+
+            // Position
+            visualElementToSetup.style.position = UiElementUtils.GetPosition(model.PositionType);
+
+            if (model.PositionTopUnit != YGUnit.YguUndefined)
+                visualElementToSetup.style.top = new Length(model.PositionTop, UiElementUtils.GetUnit(model.PositionTopUnit));
+
+            if (model.PositionBottomUnit != YGUnit.YguUndefined)
+                visualElementToSetup.style.bottom = new Length(model.PositionBottom, UiElementUtils.GetUnit(model.PositionBottomUnit));
+
+            if (model.PositionRightUnit != YGUnit.YguUndefined)
+                visualElementToSetup.style.right = new Length(model.PositionRight, UiElementUtils.GetUnit(model.PositionRightUnit));
+
+            if (model.PositionLeftUnit != YGUnit.YguUndefined)
+                visualElementToSetup.style.left = new Length(model.PositionLeft, UiElementUtils.GetUnit(model.PositionLeftUnit));
+        }
+
+        private void RemoveVisualElement(UITransformComponent uiTransformComponent)
+        {
+            if (!poolsRegistry.TryGetPool(uiTransformComponent.Transform.GetType(), out IComponentPool componentPool))
```

Comment:
`uiTransformComponent.Transform.GetType()` is redundant as you know the exact type of the Element. In other implementations it was needed because polymorphic types were supported, it's the case here.
Please check similar usages

Reply:
Done!